x/delayedack/keeper/params.go

```diff
@@ -19,3 +19,8 @@ func (k Keeper) EpochIdentifier(ctx sdk.Context) (res string) {
 	k.paramstore.Get(ctx, types.KeyEpochIdentifier, &res)
 	return
 }
+
+func (k Keeper) BridgingFee(ctx sdk.Context) (res sdk.Dec) {
```

Comment:
can we not decouple `BridgingFee` from `delayedack.Params`? Correct me if I'm wrong, but I think bridging fee params do not belong in delayedack. Injecting a `BridgingFeeKeeper` into delayedack would be cleaner IMO

Reply:
I didn't want to create additional keeper and paramspace just for this
IMO the delayedAck module is actually "IBC support for rollapps" module
which includes:
* keeper, params, and common code
* delayed ack middleware
* bridging fee
* etc..

Reply:
I do kinda agree we should decide on what this module does
It's quickly going to become a god module
I would lean on splitting things up more